src/apify/scrapy/middlewares/apify_retry.py

```diff
@@ -11,14 +11,14 @@
         'To use this module, you need to install the "scrapy" extra. Run "pip install apify[scrapy]".',
     ) from exc
 
-from ..actor import Actor
-from .utils import nested_event_loop, open_queue_with_custom_client, to_apify_request
+from ...actor import Actor
```

Comment:
Could we replace this with an absolute import? It is difficult for me to see any advantage to relatively importing something this far away.

Reply:
I understand... However, the relative imports are used everywhere in the repository. 

We can ask @jirimoravcik - is there any reason you guys preferred using relative rather than absolute imports? Thanks. 

I'd stick with that for now because of the consistency. And if there is no particular reason for that, we can replace them later in all the places. I've got no problem with that.

Reply:
I guess I already pointed this out before, huh? I agree that if we decide to replace this, we should do it in one go.

Reply:
ok, i opened an issue for that https://github.com/apify/apify-sdk-python/issues/160

Reply:
I don't remember if there was a particular reason for relative imports. I think it's fine to replace them with absolute imports. But please do it everywhere so we keep the codebase consistent.